Gemfile

```diff
@@ -23,6 +23,10 @@ end
 
 gem 'simplecov', '~> 0.8'
 
+if RUBY_VERSION < '2.0.0' || RUBY_ENGINE == 'java'
+  gem 'json', '< 2.0.0'
+end
```

Comment:
That makes sense.  Do you mind leaving a comment about it being used by simplecov?